python/test/test_github_service.py

```diff
@@ -348,6 +348,59 @@ def test_throws_exception_when_client_throws_exception(self, mock_github_client_
             ConnectionError, github_service.add_user_to_team, "test_user", 1)
 
 
+@patch("gql.transport.aiohttp.AIOHTTPTransport.__new__", new=MagicMock)
+@patch("gql.Client.__new__")
+@patch("github.Github.__new__")
+class TestGithubServiceAddUserToTeam(unittest.TestCase):
+
+    def __create_user(self, name: str) -> dict[str, str]:
```

Comment:
Is return type for this function dict[str]

Reply:
I thought it was data type, thus dict[str, int] if the second value in the dict was a number